en/docs/cli/ls.md

```diff
@@ -4,18 +4,32 @@ guide: docs_cli
 layout: guide
 ---
 
-<p class="lead">List installed packages.</p>
+<p class="lead">List all dependencies installed for the current working directory</p>
```

Comment:
I added it in case yarn decided to add support for a `--global` flag in the future, differentiating the default behavior from a global listing. I could also see someone very new not understanding `ls` is relative to the cwd and the directory's package.json / bower.json / yarn.json. I think without the 'cwd' clause, it assumes the user has some knowledge about `ls` and basic unix commands.